bootstrap.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+AUTOCONF_FILES="Makefile.in aclocal.m4 ar-lib autom4te.cache compile \
+        config.guess config.h.in config.sub configure depcomp install-sh \
+        ltmain.sh missing *libtool test-driver"
+
+case $1 in
```

Comment:
Without an argument, this basically does the equivalent of ./autogen.sh which will be confusion for users.

Reply:
We could, but I am still trying to understand what this does and why this is need

Reply:
Seems like bootstrap.sh is prefered over autogen.sh in https://www.sourceware.org/autobook/autobook/autobook_43.html

As for where/how this is called in bitbake... still trying to find it.